docs/administration/Configuration.md

```diff
@@ -515,7 +515,7 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
 
 - **Unit**: -
 - **Default**: 3
-- **Description**: The `default_replication_number` parameter is a system variable that sets the default number of replicas for each data partition when creating a table in StarRocks. The parameter can be set globally by using `set global default_replication_num=x`, where x is a positive integer. This will affect all subsequent table creations in all sessions, unless overridden by a session-level setting or a table-level setting. To set the parameter for the current session use `set default_replication_number=x`. This will affect only the current session and override the global setting, unless overridden by a table-level setting. Both global and session settings can be overridden by specifying the `replication_num=x` in the CREATE TABLE DDL.
+- **Description**: `default_replication_number` sets the default number of replicas for each data partition when creating a table in StarRocks. This setting can be overridden by specifying `replication_num=x` in the CREATE TABLE DDL.
```

Comment:
1. need to change the parameter name to "default_replication_num", not "number".  2. I think this para is not used when creating a table. the one used in creating table is a property "replication_num", for example, 
![image](https://github.com/StarRocks/starrocks/assets/98087056/98cfe3d6-1e45-462f-946b-0dd0598fc285)

Reply:
how about: Sets the default number of replicas for each data partition. This setting can be overridden by the `replication_num` property when creating a table in StarRocks. This setting takes effect if the `replication_num` property is not specified. We recommend that you set this parameter to a value less than or equal to the actual number of BE nodes in the cluster.